### PR TITLE
sanjanasarathy-hw01

### PR DESCRIPTION
**Pull request by sanjanasarathy**
While I eventually got the hang of R studio and markdown, I'm still a little confused about how the different components piece together, and more generally the workflow. After forking the repo and cloning it onto my R studio, I was able to use the markdown guide to write up my bio and include pictures and links. After previewing the work, I realized that my bullet points, pictures, and links were not working due to syntax errors of not adding spaces.

I was a little confused on how to save and then push my work back to the github page but was able to get help from a friend and by googling. I essentially saved my work as a .md file, and used the commit button on the git tab or R studio. I walked through the instructions and finally pushed the file back to my repo on GitHub. On my GitHub page, the # were still visible next to the headers so I went back and added spaces and recommitted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
-# hw01
```
Comment (hwsl17) [inline README.md:1]: Evaluation HW-01-Sanjana Sarathy
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |           |    X          |            |
| **Ease of access**          |     X      |              |            |

Remarks:

Great work Sanjana. You used all five of the elements for your biography and used them to great effect. I like the way you modified headers to differentiate sizes of the main title of your document as well as the sub-sections in a coherent manner. The coding also looks very neat-evenly spaced out. ㅓJust as a thought about comments you made, the spacing aspect of markdown is really important, and from my experience of Rmarkdowns, this is also true. Also, before you load it up on Github page, there is a way to check how your html output would look, see here for instructions. But to be honest, I find it’s easier to check on Github than going through the procedure of producing an html preview. That might come more useful for other outputs.

**Review by hwsl17 — COMMENTED** [review]
Overall fantastic work.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
-Repository for homework 1
+# Sanjana Sarathy
+
+### General Information
+I am a 4th year Public Policy major and Statistics minor in the College. I'm origninally from **Northern California**, but I am going to be moving to **New York** this fall.
+
+### Travel
+I really enjoy traveling. Last year, I was able to study abroad in Vienna!
+
+![Picture from Vienna](https://scontent.ford1-1.fna.fbcdn.net/v/t1.0-9/14485139_1286408768070500_1414751816548289039_n.jpg?_nc_cat=0&oh=15be9f29e5d1031d90a4806c8b58ba60&oe=5B760227)
```
Comment (nnickels) [inline README.md:9]: Nice! You can also insert pictures inline by saving jpg's in your repository, and calling the file path from there.

**Review by nnickels — COMMENTED** [review]
Evaluation HW-01 for Sanjana Sarathy
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X     |              |            |
| **Coding strategy**         |    X      |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |           |      X        |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Great job on the first homework! You use multiple markdown elements including images and links.
* It sounds like you did a good job thinking through some of the things you struggled with; moving forward working with .Rmd documents, you can knit your documents within RStudio to check how things look so far.

**Pull request closed without merging**